docs/snapshot-single-delete/02-call-chain.md

```
@@ -0,0 +1,47 @@
+# 02 — 处理链路总览
+
+## 2.1 快照组删除链路
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)